app/src/main/java/com/mobiledevelopmentfoodapp/dto/cart

```diff
@@ -0,0 +1,11 @@
+package com.mobiledevelopmentfoodapp.dto
+
+//import androidx.room.Entity
+//import androidx.room.PrimaryKey
+import com.google.gson.annotations.SerializedName
+
+data class Cart(var customerId: Int, var name: String, var products: ArrayList<Food>, var totalPrice: Double, @SerializedName("id") var orderId:Int = 0 ) {
+    override fun toString(): String {
+        return name
+    }
+}
```

Comment:
This looks like an incomplete feature.
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.